python/interpret_community/mimic/models/linear_model.py

```diff
@@ -160,11 +128,40 @@ def predict(self, dataset, **kwargs):
         """
         return self._linear.predict(dataset)
 
-    predict.__doc__ = (predict.__doc__ +
-                       '\nIf multiclass=True, uses the parameters for LogisticRegression:\n' +
-                       LogisticRegression.predict.__doc__.replace('-', '') +
-                       '\nOtherwise, if multiclass=False, uses the parameters for LinearRegression:\n' +
-                       LinearRegression.predict.__doc__.replace('-', ''))
+    def explain_global(self, **kwargs):
+        """Explain the model globally by either using the coef or aggregating local explanations to global.
```

Comment:
Doesn't look like this does local aggregation right now

Reply:
I see that it's in the child classes further down, but I'm not sure that it should say anything about aggregation in this particular docstring

Reply:
added aggregation